devices/MS5611/CalibrationData.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.MS5611
```

Comment:
You still have to rename the namespace here :-)

Reply:
Oh, and not only here :P Resolved